### PR TITLE
Add RGB UIColor convenience initializer

### PR DESCRIPTION
**Pull request by 3lvis**
Fixes https://github.com/UseSweet/SweetUIKit/issues/62

### CODE REVIEW

File: Sources/UIColor+Sweet.swift [hunk 1]
```diff
@@ -21,6 +21,17 @@ public extension UIColor {
...
     }
 
+    /// Convenience initializers for RGB colors.
+    ///
+    /// - Parameters:
+    ///   - red: The red part.
+    ///   - green: The green part.
+    ///   - blue: The blue part.
+    ///   - alpha: The alpha part.
+    public convenience init(r red: Double, g green: Double, b blue: Double, a alpha: Double = 100) {
```
Comment (elland) [inline Sources/UIColor+Sweet.swift:31]: From both the method signature and from documentation this looks exactly the same as the regular RGBA init method, that uses percentage instead. 🤔

Comment (3lvis) [reply Sources/UIColor+Sweet.swift:31]: That's what it is. Do you have any suggestions for naming?

Comment (elland) [reply Sources/UIColor+Sweet.swift:31]: This one uses percentages? I thought from the implementation that it ranged from 0-255, like CSS etc. 🤔 No good ideas regarding the method name so far, but documentation could clarify, including by adding the range information for the parameters. 😄

Comment (3lvis) [reply Sources/UIColor+Sweet.swift:31]: The idea is that you should be able to drop the RGB values from here:

915d2fd0-db3e-11e6-83c4-299e227b35dd

Comment (elland) [reply Sources/UIColor+Sweet.swift:31]: >  ///   - green: The green part.

becomes

>  ///   - green: The amount of green. Ranging from 0 to 255.

?

**Comment by 3lvis** [comment]
Done ✅

**Comment by elland** [comment]
Thought of a way to make the signature clearer:

```swift
public static fromRGBA(r red: Double, g green: Double, b blue: Double, a alpha: Double = 100)
// or
public static rgba(r red: Double, g green: Double, b blue: Double, a alpha: Double = 100)
```

What do you think?

**Comment by elland** [comment]
Otherwise just 🐑 it. 🚀

**Comment by 3lvis** [comment]
I think is nice to keep it as an initalizer.

**Pull request merged**